parsel/selector.py

```diff
@@ -200,12 +200,15 @@ def re_first(
             return el
         return default
 
-    def getall(self) -> List[str]:
+    def getall(self, strip: bool = False) -> List[str]:
```

Comment:
Same on `get`.

Reply:
Should we apply this to `typing.overload` of `get` as well?

Reply:
I tried adding it to `get` but then this would require some type check on default entry to validate it is a `str`, either way the behavior looks fine without it for `get`